lib/os.js

```diff
@@ -119,13 +119,13 @@ function loadavg() {
  * logical CPU core.
  *
  * @returns {Array<{
- *  model: string
- *  speed: number
+ *  model: string,
+ *  speed: number,
```

Comment:
I have a personal preference of using `;` in type definitions, but I'm fine with `,` if we already use it elsewhere.